docs/devel/CONTRIBUTING.md

```diff
@@ -299,6 +299,12 @@ notice that the container images will be pushed to
 https://github.com/inspektor-gadget/inspektor-gadget/pkgs/container/inspektor-gadget and those
 should be manually cleaned up.
 
+## Getting Help
+
+If you are having any issues with your contribution, or want to discuss anything about it, feel free
+to reach out to us on [Slack](../../README.md#slack) or on our [community
+meeting](../../README.md#community-meeting)
```

Comment:
Relative links outside of `docs/` will not work on the website https://inspektor-gadget.io/docs.

Reply:
Right. Updated to use absolute links to slack and the meeting notes instead.